doc/build-windows.md

```diff
@@ -5,7 +5,7 @@ Below are some notes on how to build Bitcoin Core for Windows.
 
 The options known to work for building Bitcoin Core on Windows are:
 
-* On Linux using the [Mingw-w64](https://mingw-w64.org/doku.php) cross compiler tool chain. Ubuntu Trusty 14.04 is recommended
+* On Linux using the [Mingw-w64](https://mingw-w64.org/doku.php) cross compiler tool chain. Ubuntu Bionic 18.04 is recommended
```

Comment:
Please add that these steps only apply for `18.04`. Only saying that it is recommended is not enough, as it means you'd still have to warn people against using `16.04`.

Reply:
Change `recommended` to `required`